src/signals/incident-management/containers/DashboardContainer/components/CategoryChart/style.scss

```diff
@@ -0,0 +1,3 @@
+.category-graph {
```

Comment:
This className is not defined anywhere, should it be category-chart?

Reply:
gedaan